frontend/build-configs/nuernberg/localization.ts

```diff
@@ -24,11 +24,18 @@ const localization: LocalizationType = {
         moreActions: {
             applyForAnotherCardTitle: "Nürnberg-Pass beantragen oder verlängern",
             applyForAnotherCardDescription: "Ihr hinterlegter Pass bleibt erhalten.",
-            activateAnotherCardTitle: "Anderen Aktivierungscode einscannen",
-            activateAnotherCardDescription: "Dadurch wird der hinterlegte Pass vom Gerät gelöscht.",
+            activateAnotherCardTitle: "Weiteren Nürnberg-Pass hinzufügen",
+            activateAnotherCardDescription: "Ihr hinterlegter Pass bleibt erhalten. Sie können diesen manuell entfernen.",
+            activationLimitDescription: "Um einen weiteren Pass hinzuzufügen, müssen Sie zuerst einen der vorhandenen Pässe löschen.",
             verifyTitle: "Einen Nürnberg-Pass prüfen",
             verifyDescription: "Prüfen Sie die Gültigkeit eines Nürnberg-Passes.",
+            removeCardTitle: "Diesen Nürnberg-Pass löschen",
+            removeCardDescription: "Nach der Auswahl wird der hinterlegte Pass vom Gerät gelöscht.",
         },
+        removeCardDialog: {
+            title: "Diese Karte löschen?",
+            description: "Wenn diese Karte gelöscht wird, muss sie vor der Wiederverwendung neu aktiviert werden.",
```

Comment:
```suggestion
            title: "Diesen Pass löschen?",
            description: "Wenn dieser Pass gelöscht wird, muss er vor der Wiederverwendung neu aktiviert werden.",
```

I am also not super happy with the term `Wiederverwendung`. How about something like:
`Wenn dieser Pass gelöscht wird, muss dieser für eine erneute Verwendung neu hinzugefügt werden`

Reply:
ok i just took the suggestions from ui/ux

Reply:
rename and also adjusted for bayern